app/api.js

```diff
@@ -408,8 +408,33 @@ export function getPeersWithObjectbyHash (hash) {
  */
 export function importObjectByHash (hash) {
   if (!IPFS_CLIENT) return Promise.reject(ERROR_IPFS_UNAVAILABLE)
+  const uuid = uuidv4()
+
+  app.emit('new-activity', {
+    uuid,
+    hash,
+    type: 'import-from-hash',
```

Comment:
Do we want to make the Type as a Variable?

Reply:
Something like this?
```
const activityTypes = {
  ADD: 'add',
  IMPORT_FROM_HASH: 'import-from-hash'
}
```

Reply:
yes that looks good to me :) So we can test this easier later on